plugin/evm/ftso_snapshot.go

```diff
@@ -22,6 +22,35 @@ type FTSOSnapshot struct {
 	contracts FTSOContracts
 }
 
+func (f *FTSOSnapshot) Cap() (float64, error) {
+
+	supply := &big.Int{}
+	err := BindEVM(f.system.blockchain).
+		AtBlock(f.start).
+		OnContract(f.contracts.WNAT).
+		Execute(TotalSupply).
+		Decode(&supply)
+	if err != nil {
+		return 0, fmt.Errorf("could not get total supply: %w", err)
+	}
+
+	fraction := &big.Int{}
+	err = BindEVM(f.system.blockchain).
+		AtBlock(f.start).
+		OnContract(f.contracts.Manager).
+		Execute(Settings).
+		Decode(&fraction, nil, nil, nil, nil, nil, nil, nil, nil, nil)
```

Comment:
"settings" function returns 9 parameters 
https://songbird-explorer.flare.network/address/0xbfA12e4E1411B62EdA8B035d71735667422A6A9e/contracts

```suggestion
		Decode(&fraction, nil, nil, nil, nil, nil, nil, nil, nil)
```